utils/listener.py

```diff
@@ -0,0 +1,5 @@
+bad_words = ['fuck', 'ass', 'penis', 'shit']
+
+
+async def check_for_profanity(message):
+    return any([bad_word in message.content.lower().split() for bad_word in bad_words])
```

Comment:
Will this code detect a bad word regardless of whether it's typed in upper or lower case? Or does it only work for lowercase words that match exactly those in the list?

Reply:
You know what I just looked again and realized it does do that. Disregard my comment above.